### PR TITLE
Update COVID Dashboard README

### PR DESCRIPTION
**Pull request by edumorlom**
Updates to README include:
1. Adds steps on deploying.
2. Adds steps on running locally.
3. References Social Wellness dashboard.
4. Explains how to add new columns/categories to Table.
5. Changes to APIs have been reflected.

### CODE REVIEW

File: covid19-dashboard/README.md [hunk 1]
```diff
@@ -62,41 +55,102 @@ in the US with NYT COVID-19 data.
...
+gcloud config set project $GCP_PROJECT && \
+gcloud auth application-default login
+```
+
+#### Install Node.JS 12 and Python 3.7
+
+**NOTE:** Although other versoins of Node might work, only Node 12 has been tested.
+**NOTE:** Although other versions of Python might work, only Python 3.7 has been tested.
+
+Before running the server, install [nodejs](https://nodejs.org/en/download/)
```
Comment (beets) [inline covid19-dashboard/README.md:136]: nit: repetitive - use bullets?

File: covid19-dashboard/README.md [hunk 2]
```diff
@@ -62,41 +55,102 @@ in the US with NYT COVID-19 data.
...
+```
+
+## Local Development
+
+### Set Google Application Credential
+
+**NOTE:** This project is currently deployed under the GCP Project `datcom-wesbite`, but any other project can be used instead.
+
+```bash
+export GOOGLE_CLOUD_PROJECT="datcom-website"
```
Comment (beets) [inline covid19-dashboard/README.md:126]: include this in the script for those of us who jump around multiple projects?
see the run_server.sh script in the website repo for an example

File: covid19-dashboard/README.md [hunk 3]
```diff
@@ -62,41 +55,102 @@ in the US with NYT COVID-19 data.
...
+gcloud config set project $GCP_PROJECT && \
+gcloud auth application-default login
+```
+
+#### Install Node.JS 12 and Python 3.7
+
+**NOTE:** Although other versoins of Node might work, only Node 12 has been tested.
+**NOTE:** Although other versions of Python might work, only Python 3.7 has been tested.
```
Comment (beets) [inline covid19-dashboard/README.md:134]: combine the notes to reduce repetition?

Comment (edumorlom) [reply covid19-dashboard/README.md:134]: fixed

File: covid19-dashboard/README.md (hunk 1 above)
Comment (edumorlom) [reply covid19-dashboard/README.md:136]: fixed

File: covid19-dashboard/README.md (hunk 2 above)
Comment (edumorlom) [reply covid19-dashboard/README.md:126]: done

**Review by beets — APPROVED** [review]

**Pull request merged**